README.md

```diff
@@ -9,21 +9,17 @@ in a directory of which the parent is added as import path.
 
 ### Installing
 
-    qmake
+    mkdir build
+    cd build
+    cmake -DCMAKE_INSTALL_PREFIX=.../lib/ ..
```

Comment:
What is `.../lib/`?

Reply:
by default, it installs in /usr/local/lib. The variable CMAKE_INSTALL_PREFIX allows to override where to install the library.

Reply:
Right, I was mainly wondering what this example was, but I guess the `.../lib/` is just a placeholder for something useful?

Reply:
yes, it's a placeholder if you need to change the default value. I can write it between `<>` or if you have a better idea (or not mentionning it at all)?

Reply:
If passing `CMAKE_INSTALL_PREFIX` is not normally required for using this library, then I think it's best not to mention it, or to only mention is separately, if it's not a standard variable packagers are already familiar with. But if it is normally required to pass something, I think the example should still be one that could be copy/pasted.

Reply:
It's a standard cmake variable only for packagers or people who want to install it so I removed it from the README.md